heron/scheduler-core/src/java/com/twitter/heron/scheduler/RuntimeManagerMain.java

```diff
@@ -318,7 +319,9 @@ public static void main(String[] args)
       // SUPPRESS CHECKSTYLE IllegalCatch
     } catch (UpdateDryRunResponse response) {
       LOG.log(Level.FINE, "Sending out dry-run response");
-      System.out.print(runtimeManagerMain.renderDryRunResponse(response));
+      // Output may contain UTF-8 characters, so we should print using UTF-8 encoding
+      PrintStream out = new PrintStream(System.out, true, "UTF-8");
```

Comment:
We can't use enum here since Java does not have such API. See: https://docs.oracle.com/javase/7/docs/api/java/io/PrintStream.html. However, I agree we can use the `.name()`.

And yes, I should allow some hours for reviewers to review. Sorry I've been slow and hurrying myself too much on dry-run.